.gitignore

```diff
@@ -3,4 +3,6 @@
 *.jl.mem
 precompile.jl
 build
-Manifest.toml
\ No newline at end of file
+Manifest.toml
+
+e2e/node_modules
```

Comment:
Can we put it in the folder `/test/frontend/` instead of `/e2e/`?

We can move the Julia tests into `/test/backend/`, but `runtests.jl` needs to stay in `/test/` to count as a Julia package